jme3-code-check/src/com/jme3/gde/codecheck/hints/InternalMethodHint.java

```diff
@@ -1,134 +1,84 @@
 package com.jme3.gde.codecheck.hints;
 
-import com.jme3.system.Annotations.Internal;
```

Comment:
I think we should open an issue on the engine to check the source and add a few of these where appropriate, as I think noone knows about them anymore :D